aws/resource_aws_lb_target_group_test.go

```diff
@@ -1702,6 +1757,56 @@ resource "aws_vpc" "test" {
 `, targetGroupName)
 }
 
+func testAccAWSLBTargetGroupConfig_GRPC_ProtocolVersion(targetGroupName string) string {
+	return fmt.Sprintf(`
+resource "aws_lb_target_group" "test" {
+  name     = "%s"
+  port     = 80
+  protocol = "HTTP"
+  protocol_version = "GRPC"
+  vpc_id   = aws_vpc.test2.id
```

Comment:
I bet the `terrafmt` error is due to the spacing here (the `=` signs aren't all in line).

Reply:
Looks like that is correct I fixed it.